dss/logging.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from logging import NOTSET, DEBUG, INFO, WARNING, ERROR, CRITICAL
+import sys
+
+from typing import Mapping, Union, Sequence, Tuple
+
+import dss
+from dss.config import Config
+
+log_level_t = Mapping[Union[None, str, logging.Logger], Tuple[int, ...]]
+"""
+Values of this type map a given logger and debug level to the log level for that logger at the given debug level. The
```

Comment:
I don't really get this section.  I think I understand how it works from the code, not from the code.

Reply:
I rewrite the section.

Reply:
I will rewrite the section.

Reply:
Rewritten.